sdk/python/kfp/dsl/pipeline_context.py

```diff
@@ -27,7 +28,8 @@ def pipeline(func: Optional[Callable] = None,
              name: Optional[str] = None,
              description: Optional[str] = None,
              pipeline_root: Optional[str] = None,
-             display_name: Optional[str] = None) -> Callable:
+             display_name: Optional[str] = None,
+             pipeline_config: pipeline_config.PipelineConfig = None) -> Callable:
```

Comment:
Optional nitpick: Add the new `pipeline_config` parameter to the docstring?

Reply:
done